fix: Global min Self Delegation (#134)
osmosis-labs/cosmos-sdk: docs/core/proto-docs.md

| `historical_entries` | [uint32](#uint32) |  | historical_entries is the number of historical entries to persist. |
| `bond_denom` | [string](#string) |  | bond_denom defines the bondable coin denomination. |
| `min_commission_rate` | [string](#string) |  | min_commission_rate is the chain-wide minimum commission rate that a validator can charge their delegators |

> | `min_self_delegation` | [string](#string) |  | min_commission_rate is the chain-wide minimum commission rate that a validator can charge their delegators |

Reviewer: Need to update the description for this one

> | `min_self_delegation` | [string](#string) |  | min_commission_rate is the chain-wide minimum commission rate that a validator can charge their delegators |

Reviewer: ```suggestion
| `min_self_delegation` | [string](#string) |  | min_self_delegation is the chain-wide minimum self delegation a validator can have |
```
